### DISCUSSION TITLE
Unable to watch and copy already existent API-key

### DISCUSSION THREAD

**Discussion by wra1w0** [simple-login/app#1121] [Q&A] [answered]
I don't see the button to watch and copy already existent API-key. The screenshot is from Firefox (v. 101.0.1), the same behaviour is in Brave (v. 1.40.107).
<img width="586" alt="image" src="https://user-images.githubusercontent.com/47259322/176386065-fcf012b1-c4c8-4762-a7c3-b9a66f54f047.png">

**Comment by ntnhon** [comment, marked as answer]
You can no longer retrieve API keys because they are now encrypted in our database for security reasons. But you can always generate a new one and copy it right after the creation.

**Reply by wra1w0** [reply]
OK. I understand, thank you. Maybe some information (e.g. banner) with this information on the API-key page will be nice.
